sw/device/silicon_creator/lib/base/static_critical.ld

```diff
@@ -19,8 +19,10 @@ INCLUDE sw/device/silicon_creator/lib/base/static_critical_size.ld
 
   KEEP(*(.static_critical.boot_measurements))
   KEEP(*(.static_critical.sec_mmio_ctx))
+  KEEP(*(.static_critical.in_memory_epmp))
```

Comment:
nit: let's keep this sorted, i.e. boot_measurements, epmp_state, sec_mmio_ctx.

Reply:
Fixed